### ISSUE TITLE
Windows Sliver Server Exe Generate Error

### ISSUE THREAD

**Issue by secfb** [BishopFox/sliver#315]
I downloaded the latest Windows version of the program. When I try to create an exe file I get the following error. Metasploit required for Windows version to work?

Command used
```generate -m 192.168.1.3 -s 1.exe -o Windows```

![1](https://user-images.githubusercontent.com/38748801/102988419-ecdd5700-4524-11eb-840b-8150b1b12830.png)

**Desktop (please complete the following information):**
 - OS: Windows 10  [Version 10.0.19042.685]
 - Sliver Version [1.2.0]
---------------------------
**Question about Linux Version**
I use the linux version in Kali operating system. Why is the exe file creation time long. It takes at least 15 minutes and the size of the exe file created is 7mb on average. Do you think this size is normal? Does it always take this long? Or am I using it wrong?

- OS: Kali Linux 5.9 Kernel
 - Sliver Version [1.2.0]
![2](https://user-images.githubusercontent.com/38748801/103002214-8d407500-453f-11eb-87c0-02e8dde9304e.png)

---------------------------

Also, your wiki page is not very clear. I wish you could make a video about the use of a tool. I could not find a proper resource either.

**Comment by lesnuages** [comment]
First issue looks like a bug.
Second question is totally expected behavior. Go compiled binaries are usually bigger than dynamically linked executables, we can't do much about that. As for the compile time, it's due to the obfuscation process. You can use the --skip-symbols flag to speed up the process, but you'll lose the symbol obfuscation.

**Comment by secfb** [comment]
I have golang installed in the operating system I am using, but I am getting the error I made in the screenshot.

Thank you very much for the information. I liked Sliver tool. I consider using it for security tests. Are you considering making a video of the wide use of Sliver.

**Comment by rkervella** [comment]
@moloch-- looks like a bug in the obfuscation process, when we move the source to the new `$GOPATH`. On Windows, looks like we're renaming the package names with backslashes instead of forward slashes (yeah, another Windows specific issue). I was able to reproduce:

```
All hackers gain haste
[*] Server v1.2.0 - 427d680d7c7d2ce2b0bd91bf9de0aad1e7b11275
[*] Welcome to the sliver shell, please type 'help' for options

sliver > generate -m 192.168.122.9 -s 1.exe -a x86

[*] Generating new windows/386 implant binary
[*] Symbol obfuscation is enabled.
[*] This process can take awhile, and consumes significant amounts of CPU/Memory
[!] rpc error: code = Unknown desc = package move: invalid move destination: fgdhlnaegd\mipgajppog; gomvpkg does not support move destinations whose base names are not valid go identifiers
```

@secfb I'd recommend using the sliver server on either Linux or MacOS in the meantime. If you absolutely want to run it on Windows, consider running it on a WSL subsystem (it usually works fine there).

**Comment by moloch--** [comment]
Oh Windows ... should be easy to fix at least, I'll look into it this weekend.

**Comment by moloch--** [comment]
Fixed in #342
